### DISCUSSION TITLE
Simple search bundle

### DISCUSSION THREAD

**Discussion by alexej-d** [pimcore/pimcore#7766] [General]
Hi, has anyone managed to create some kind of bundle for frontend search? I'm thinking of something as simple as e.g. WordPress without external search indexing services like ElasticSearch.

**Comment by dpfaffenbauer** [comment]
https://github.com/dachcom-digital/pimcore-dynamic-search

Not simple, but powerful

**Reply by alexej-d** [reply]
Thank you, I have already seen that bundle, but to my understanding you would still need an external index service like ES, which is exactly what I do not want 😅

**Reply by dpfaffenbauer** [reply]
no, it you can use Zend Lucene with it. Which is currently the only provider implemented.

**Reply by dpfaffenbauer** [reply]
Zend Lucene doesn't have external dependencies, it creates the index on the filesystem

**Reply by alexej-d** [reply]
Ok, thank you, is it also based on PHP?

**Reply by dpfaffenbauer** [reply]
yep, no need for any external service

**Reply by alexej-d** [reply]
@dpfaffenbauer I've successfully implemented the search, but I do not seem to be able to correctly get multi search to work... Setting blocks to the following renders the same results under every block:
```
blocks : 
    document:
        reference: search
    news:
        reference: search
    other_type:
        reference: search
